### PR TITLE
Various updates to operator pages

### PR DESCRIPTION
**Pull request by Fraser999**
Part of #141.

### CODE REVIEW

File: source/docs/casper/operators/bonding.md [hunk 1]
```diff
@@ -46,15 +47,15 @@ Note the following in the above command:
 - The chain name for Mainnet is `casper` and for Testnet is `casper-test`
 - The default port for node address is 7777
 - The session arguments need to be encased in double-quotes, with the parameter values in single quotes
-- The payment amount is specified in motes, it must contain at least 10 zeros
+- The payment amount is specified in motes, where 1 cspr is 1,000,000,000 motes
```
Comment (ipopescu) [inline source/docs/casper/operators/bonding.md:50]: Suggestion to use CSPR instead of cspr.

**Review by ipopescu — APPROVED** [review]
Wonderful updates. I left a minor suggestion.

**Review by GloreenS — APPROVED** [review]
The changes look good. I added a commit for the change to the command discussed yesterday.

File: source/docs/casper/operators/bonding.md (hunk 1 above)
Comment (Fraser999) [reply source/docs/casper/operators/bonding.md]: Done in [`0185262d`](https://github.com/casper-network/docs/pull/180/commits/0185262d28719ab5c31c4cfb3a79f98665a2da01).

**Pull request merged**